x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/statistic_card.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { EuiCard, EuiFlexItem, EuiLink, EuiTitle } from '@elastic/eui';
+
+import { useRoutes } from '../shared/use_routes';
+
+interface IStatisticCardProps {
+  title: string;
+  count?: number;
+  actionPath?: string;
+}
+
+export const StatisticCard: React.FC<IStatisticCardProps> = ({ title, count = 0, actionPath }) => {
+  const { getWSRoute } = useRoutes();
+
+  const card = (
+    <EuiFlexItem>
+      <EuiCard
+        className={actionPath ? 'euiCard--isClickable' : ''}
+        layout="horizontal"
+        title={title}
+        titleSize="xs"
+        description={
+          <EuiTitle size="l">
+            <span>
+              <span className={`euiTextColor euiTextColor--${actionPath ? 'primary' : 'subdued'}`}>
+                {count}
+              </span>
+            </span>
+          </EuiTitle>
+        }
+      />
+    </EuiFlexItem>
+  );
+
+  return (
+    <EuiFlexItem>
+      {actionPath ? (
+        <EuiLink target="_blank" href={getWSRoute(actionPath)}>
+          {card}
+        </EuiLink>
```

Comment:
^ As above, I'd like to see us simplify this by using the `href` prop on EuiCard instead of wrapping it in an EuiLink.

Eventually (once this link destination is managed by React Router) we'll want to create an [EuiReactRouterCard helper](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/shared/react_router_helpers/eui_link.tsx#L53) where we pass a `to={path}` prop and it handles both href & onClick behavior for us, but that should hopefully be fairly straightforward.

Reply:
7ba1184